src/test/java/frc/InitTests.java

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc;
+
+import edu.wpi.first.hal.HAL;
+import frc.robot.Robot;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+/** Test robot init. */
+public class InitTests {
+  @BeforeEach
+  private void setup() {
+    HAL.initialize(500, 0);
+  }
+
+  @Test
+  private void initTest() {
```

Comment:
Prefer test names that say what is expected. These are never called, so you should be excessively verbose since it's what's going to communicate intent when it fails. F.e, "robotShouldNotCombustWhenFedFrechfries()" would be a totally reasonable unit test method name.

Reply:
Also note you don't event need test in the name - you've already communicated it's a test with the annotation

Reply:
All that being said, what's your goal with this method? When does it fail? What does it tell you when it does?

Reply:
the goal here is just to make sure `Robot` initializes without throwing an error, mainly as a simple template for unit tests as we (hopefully) add more throughout the season

Reply:
Then I'd make it explicit with a [assertDoesNotThrow](https://junit.org/junit5/docs/current/api/org.junit.jupiter.api/org/junit/jupiter/api/Assertions.html#assertDoesNotThrow(org.junit.jupiter.api.function.Executable)). Communicates your intent and makes clear you didn't forget the assert along the way.